src/Module/Post/Edit.php

```diff
@@ -0,0 +1,228 @@
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Module\Post;
+
+use Friendica\App;
+use Friendica\BaseModule;
+use Friendica\Content\Feature;
+use Friendica\Core\Hook;
+use Friendica\Core\L10n;
+use Friendica\Core\Renderer;
+use Friendica\Core\Session\Capability\IHandleUserSessions;
+use Friendica\Model\Contact;
+use Friendica\Model\Post;
+use Friendica\Model\User;
+use Friendica\Module\Response;
+use Friendica\Navigation\SystemMessages;
+use Friendica\Network\HTTPException;
+use Friendica\Util\Crypto;
+use Friendica\Util\Profiler;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Controller to edit a post
+ */
+class Edit extends BaseModule
+{
+	/** @var IHandleUserSessions */
+	protected $session;
+	/** @var SystemMessages */
+	protected $sysMessages;
+	/** @var App\Page */
+	protected $page;
+	/** @var App\Mode */
+	protected $mode;
+	/** @var App */
+	protected $app;
+	/** @var bool */
+	protected $isModal = false;
+
+	public function __construct(L10n $l10n, App\BaseURL $baseUrl, App\Arguments $args, LoggerInterface $logger, Profiler $profiler, Response $response, IHandleUserSessions $session, SystemMessages $sysMessages, App\Page $page, App\Mode $mode, App $app, array $server, array $parameters = [])
+	{
+		parent::__construct($l10n, $baseUrl, $args, $logger, $profiler, $response, $server, $parameters);
+
+		$this->session     = $session;
+		$this->sysMessages = $sysMessages;
+		$this->page        = $page;
+		$this->mode        = $mode;
+		$this->app         = $app;
+	}
+
+
+	protected function content(array $request = []): string
+	{
+		$this->isModal = $request['mode'] ?? '' === 'none';
+
+		if (!$this->session->getLocalUserId()) {
+			$this->errorExit($this->t('Permission denied.'), HTTPException\UnauthorizedException::class);
+		}
+
+		$postId = $this->parameters['post_id'];
+
+		if (empty($postId)) {
+			$this->errorExit($this->t('Post not found.'), HTTPException\BadRequestException::class);
+		}
+
+		$fields = ['allow_cid', 'allow_gid', 'deny_cid', 'deny_gid',
+				   'body', 'title', 'uri-id', 'wall', 'post-type', 'guid'];
+
+		$item = Post::selectFirstForUser($this->session->getLocalUserId(), $fields, [
+			'id'  => $postId,
+			'uid' => $this->session->getLocalUserId(),
+		]);
+
+		if (empty($item)) {
+			$this->errorExit($this->t('Post not found.'), HTTPException\BadRequestException::class);
+		}
+
+		$user   = User::getById($this->session->getLocalUserId());
+		$geoTag = '';
+
+		$output = Renderer::replaceMacros(Renderer::getMarkupTemplate('section_title.tpl'), [
+			'$title' => $this->t('Edit post'),
+		]);
+
+		$this->page['htmlhead'] .= Renderer::replaceMacros(Renderer::getMarkupTemplate('jot-header.tpl'), [
+			'$ispublic'  => '&nbsp;', // $this->t('Visible to <strong>everybody</strong>'),
+			'$geotag'    => $geoTag,
+			'$nickname'  => $this->app->getLoggedInUserNickname(),
+			'$is_mobile' => $this->mode->isMobile(),
+		]);
```

Comment:
The variable `$geoTag` is never used. Also what is the comment for the `$ispublic` template variable?
```suggestion
		$output = Renderer::replaceMacros(Renderer::getMarkupTemplate('section_title.tpl'), [
			'$title' => $this->t('Edit post'),
		]);

		$this->page['htmlhead'] .= Renderer::replaceMacros(Renderer::getMarkupTemplate('jot-header.tpl'), [
			'$ispublic'  => '&nbsp;', // $this->t('Visible to <strong>everybody</strong>'),
			'$geotag'    => '',
			'$nickname'  => $this->app->getLoggedInUserNickname(),
			'$is_mobile' => $this->mode->isMobile(),
		]);
```